### PR TITLE
[RHPAM-3123] userTaskService.saveContent doesn't work

### PR DESCRIPTION
**Pull request by elguardian**
jira
https://issues.redhat.com/browse/RHPAM-3123

### CODE REVIEW

**Comment by elguardian** [comment]
Jenkins retest this please.

**Comment by elguardian** [comment]
Jenkins run fdb

**Comment by elguardian** [comment]
Jenkins run fdb

**Comment by elguardian** [comment]
Jenkins run fdb

File: jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/UserTaskServiceImplTest.java [hunk 1]
```diff
@@ -893,7 +893,7 @@ public void testTaskOperationWithUserOrWithIdentityProvider() {
         assertEquals(TaskEvent.TaskEventType.STOPPED, auditTasks.get(3).getType());
         assertEquals("salaboy", auditTasks.get(3).getUserId());
         assertEquals(TaskEvent.TaskEventType.UPDATED, auditTasks.get(4).getType());
-        assertEquals("salaboy", auditTasks.get(4).getUserId());
+        assertEquals("Administrator", auditTasks.get(4).getUserId());
```
Comment (afalhambra) [inline jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/UserTaskServiceImplTest.java:896]: Is this expected behavior? I mean, after making a call to `saveContent`, task owner changes from current user `salaboy` to `Administrator` (or whoever it's set in the `org.jbpm.ht.admin.user` property). Is this ok or am I missing anything here?

File: jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/UserTaskServiceImpl.java [hunk 2]
```diff
@@ -700,7 +700,8 @@ public Long saveContentFromUser(Long taskId, String userId, Map<String, Object>
 
     @Override
     public Long saveContent(String deploymentId, Long taskId, Map<String, Object> values) {
-        return saveContent(deploymentId, taskId, null, values);
+        String userId = System.getProperty("org.jbpm.ht.admin.user", "Administrator");
```
Comment (afalhambra) [inline jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/UserTaskServiceImpl.java:703]: not sure why an admin user is required here to saveContent? shouldn't this user be fetched from `userGroupCallback` instead? can you please clarify?

**Review by afalhambra — COMMENTED** [review]
Just some questions about task ownership

**Pull request closed without merging**